config/db.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const connectToDbAndStartServer = (server) => {
```

Comment:
This function should only connect to database. Starting server should be in the index file.